enterprise/server/cmd/goinit/main.go

```diff
@@ -98,8 +122,13 @@ func main() {
 		fmt.Printf("Error configuring logging: %s", err)
 		os.Exit(1) // in case log.Fatalf does not work.
 	}
+	log.Printf("os.Args before: %q", os.Args)
```

Comment:
did you mean to leave these log statements in?

Reply:
Nope, done.